hydra/helpers/client.py

```diff
@@ -153,10 +153,14 @@ def bootstrap(self, destination, version=None, destroy=False):
         self.app.log.info('Initializing Loom...')
 
         self.app.utils.binary_exec('./shipchain', 'init')
-        node_key = self.app.utils.binary_exec('./shipchain', 'nodekey').stdout.strip()
 
-        time.sleep(1)  # Gotta wait a second because the priv_validator doesn't always show up
+        self.update_node_helper_files(version)
+
+        self.app.log.info('Bootstrapped!')
 
+    def update_node_helper_files(self, version):
+        node_key = self.app.utils.binary_exec('./shipchain', 'nodekey').stdout.strip()
+        time.sleep(1)  # Gotta wait a second because the priv_validator doesn't always show up
```

Comment:
I think this should stay inside the `bootstrap()` function; I think this is waiting on the `./shipchain init` to generate the initial keys. Shouldn't be needed with `configure()`